aten/src/ATen/templates/LazyIr.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+// This file contains autogenerated LazyTensor IR nodes
+${lazy_ir_sysinc}
+${lazy_ir_inc}
+
+namespace torch {
+namespace lazy {
+using at::operator<<;
```

Comment:
this is probably cargo culted but could you file an issue about making sure ADL actually works for `operator<<` in all cases. This is probably a case of someone sticking an `operator<<` overload in the wrong namespace

Reply:
It might be possible that we only had this issue back when LazyIr.h was using its own namespace (not torch::lazy).  I will check if it can be removed now.  But you might still want me to file that issue since at least it was a problem in our old namespace setup.